src/index.ts

```diff
@@ -108,8 +134,42 @@ function mouseMoving(event: MouseEvent) {
 }
 
 /**
- * Cancels the mouseMoving function after the mouse has been released.
+ * Determines the original place of an atom with a mouse click.
+ * @param event The action of a user click
  */
-function stopListening() {
-    document.getElementById("ellipses")?.removeEventListener("mousemove", mouseMoving);
+function atomLocation(event: MouseEvent) {
+    startingPoint = {
+        x: event.clientX - xshift,
+        y: event.clientY - yshift,
+    };
+    document.addEventListener("keypress", drawAtom);
+}
+
+/**
+ * Determines what the atom will look like, and places it in the original location.
+ * When the mouse is clicked again it places the atom and it cannot be moved.
+ * @param event The action of selecting a letter
+ */
+function drawAtom(event: KeyboardEvent) {
+    atom = event.key;
+    ctx.fillText(atom, startingPoint.x, startingPoint.y);
+    ctx.stroke;
+    canvas.addEventListener("mousemove", atomMoving);
+    canvas.addEventListener("mousedown", () => {
+        canvas.removeEventListener("mousemove", atomMoving);
+    });
+}
+
+/**
+ * When the mouse is moved it places the atom directly under it until in the desired location.
+ * @param event The event of the mouse moving
+ */
+function atomMoving(event: MouseEvent) {
+    ctx.clearRect(0, 0, canvas.width, canvas.height);
+    const currentPoint: Point = {
+        x: event.clientX - xshift,
+        y: event.clientY - yshift,
+    };
+    ctx.fillText(atom, currentPoint.x, currentPoint.y);
+    ctx.stroke();
```

Comment:
Not sure how i feel about the position of the mouse in relation to the text in bottom left, could be better in center (probably not since i want to be able to see the letter), or center bottom maybe?